Openshift/supporting-images/nexus-with-entando-dependencies/maven-setup/settings.xml

```diff
@@ -19,9 +19,9 @@
    <!-- ### configured mirrors ### -->
    <mirror>
      <id>mirror.default</id>
-     <mirrorOf>*</mirrorOf>
+     <mirrorOf>nothing</mirrorOf>
      <name>An internal openshift mirror</name>
-     <url>http://localhost:8081/repository/maven-public/</url>
+     <url>http://172.17.0.3:8081/repository/maven-public/</url>
```

Comment:
What does this IP point to?

Reply:
It is just an optimization for local development - this would be the ip where you have your own nexus running. I turn this mirror off before I check in (<mirrorOf>nothing</mirrorOf>).